q2_feature_table/_examples.py

```diff
@@ -0,0 +1,357 @@
+        Table(np.array([[0, 2, 6], [2, 2, 4]]),
+              ['O1', 'O3'],
+              ['S4', 'S5', 'S6']))
+
+
+def ft3_factory():
+    return Artifact.import_data(
+        'FeatureTable[Frequency]',
+        Table(np.array([[0, 4, 9], [4, 4, 8]]),
+              ['O1', 'O4'],
+              ['S7', 'S8', 'S9']))
+
+
+def feature_table_merge_example(use):
+    feature_table1 = use.init_artifact('feature_table1', ft1_factory)
+    feature_table2 = use.init_artifact('feature_table2', ft2_factory)
+
+    merged_table, = use.action(
+        use.UsageAction(plugin_id='feature_table',
+                        action_id='merge'),
+        use.UsageInputs(tables=[feature_table1, feature_table2]),
+        use.UsageOutputNames(merged_table='merged_table'),
+    )
+
+
+def feature_table_merge_three_tables_example(use):
+    feature_table1 = use.init_artifact('feature_table1', ft1_factory)
+    feature_table2 = use.init_artifact('feature_table2', ft2_factory)
+    feature_table3 = use.init_artifact('feature_table3', ft3_factory)
+
+    merged_table, = use.action(
+        use.UsageAction(plugin_id='feature_table',
+                        action_id='merge'),
+        use.UsageInputs(
+            tables=[feature_table1, feature_table2, feature_table3],
+            overlap_method='sum'
+        ),
+        use.UsageOutputNames(merged_table='merged_table'),
+    )
+
+
+def feature_table_merge_seqs(use):
+    seqs1 = use.init_artifact_from_url('seqs1', rep_seqs_1_url)
+    seqs2 = \
+        use.init_artifact_from_url('seqs2', rep_seqs_2_url)
+
+    merged_data, = use.action(
+        use.UsageAction('feature_table', 'merge_seqs'),
+        use.UsageInputs(
+            data=[seqs1, seqs2]
+        ),
+        use.UsageOutputNames(
+            merged_data='merged_data'
+        )
+    )
+
+
+def feature_table_filter_seqs_metadata1(use):
+    seqs = use.init_artifact_from_url(
+        'seqs', rep_seqs_1_url
+    )
+    sample_metadata = use.init_metadata_from_url(
+        'sample_metadata', moving_pics_md_url
+    )
+
+    filtered_seqs, = use.action(
+        use.UsageAction(plugin_id='feature_table', action_id='filter_seqs'),
+        use.UsageInputs(table=seqs, metadata=sample_metadata,
+                        where='[subject]="subject-1"'),
+        use.UsageOutputNames(filtered_seqs='filtered_seqs')
+    )
+
+
+def feature_table_filter_seqs_metadata2(use):
+    feature_table = use.init_artifact_from_url(
+        'feature_table', rep_seqs_1_url
+    )
+    sample_metadata = use.init_metadata_from_url(
+        'sample_metadata', moving_pics_md_url
+    )
+
+    filtered_seqs, = use.action(
+        use.UsageAction(plugin_id='feature_table', action_id='filter_seqs'),
+        use.UsageInputs(table=feature_table, metadata=sample_metadata,
+                        where='[body-site] IN ("left palm", "right palm")'),
+        use.UsageOutputNames(filtered_seqs='filtered_seqs')
+    )
+
+
+def feature_table_filter_seqs_metadata3(use):
```

Comment:
```suggestion
def feature_table_filter_seqs_subject_and_bodysite(use):
```
Same as above comments re: naming convention.